app/src/main/java/com/project/jangburich/ui/home/HomeFragment.kt

```
@@ -0,0 +1,60 @@
+    fun initView() {
+        binding.run {
+            toolbar.run {
+                buttonWallet.setOnClickListener {
+                    // 나의 지갑 화면
+                }
+            }
+        }
+    }
+
+    override fun onButtonClicked() {
+    }
```

detekt

[warning] This empty block of code can be removed.

(detekt.empty-blocks.EmptyFunctionBlock)